plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4598,6 +4615,27 @@ public void adjust(Mechanism mechanism) {
             state.update();
         }
 
+        // <--[mechanism]
+        // @object LocationTag
+        // @name waxed
+        // @input ElementTag(Boolean)
+        // @description
+        // Changes whether the sign at the location is waxed.
+        // @tags
+        // <LocationTag.waxed>
+        // -->
+        if (mechanism.matches("waxed") && mechanism.requireBoolean()) {
+            BlockState state = getBlockState();
+            if (!(state instanceof Sign)) {
+                mechanism.echoError("'waxed' mechanism can only be called on Sign blocks.");
+            }
+            else {
+                Sign sign = (Sign) state;
+                sign.setWaxed(mechanism.getValue().asBoolean());
+                sign.update();
+            }
```

Comment:
Should probably use the same early-return pattern as the tag - `if !sign return, set waxed`